examples/integrated_gradients_imdb.ipynb

```diff
@@ -19,7 +19,7 @@
   },
```

Comment:
<p>Can leave out <code>compute_layer_input_gradients=False</code>  as that is the default unless there's another reason to keep it explicit here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/417/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
I think it's better to keep it explicitly since it's an embedding layer and you cannot run it for inputs